### DISCUSSION TITLE
Sending messages at low level

### DISCUSSION THREAD

**Discussion by slhermit** [fetchai/cosmpy#322] [Developers]
Hello,

I am trying to send messages to the Osmo blockchain. However, the low-level-api.md doc still refers to built-in proto messages (like MsgGrant below). Would you have a code example on how a raw JSON message could be signed and sent ? 

Thx for the help

msg = MsgGrant(
    granter=str(validator.address()),
    grantee=str(wallet.address()),
    grant=grant,
)

tx = Transaction()
tx.add_message(msg)

tx = prepare_and_broadcast_basic_transaction(client, tx, validator)

**Comment by johnqh** [comment]
Pump
